storage.bs

```diff
@@ -228,24 +266,14 @@ given a <a for=map>value</a> when a <a>storage shelf</a> is
 <var>type</var>, run these steps:
 
 <ol>
- <li><p>Let <var>key</var> be <var>environment</var>'s
- <a for="environment settings object">origin</a>.
-
- <li><p>If <var>key</var> is an <a>opaque origin</a>, then return failure.
-
- <li><p>If the user has disabled storage, then return failure.
-
- <li>
-  <p>If <var>shed</var>[<var>key</var>] does not <a for=map>exist</a>, then:
-
-  <ol>
-   <li><p>Let <var>shelf</var> be a new <a>storage shelf</a>.
+ <li><p>Let <var>key</var> be the result of running <a>obtain a storage key</a> with
+ <var>environment</var>.
 
-   <li><p>Set <var>shelf</var>'s <a>bucket map</a>["<code>default</code>"] to the result of
-   <a>create a storage bucket</a> with <var>type</var>.
+ <li><p>If <var>key</var> is failure, then return failure.
 
-   <li><p>Set <var>shed</var>[<var>key</var>] to <var>shelf</var>.
-  </ol>
+ <li><p>If <var>shed</var>[<var>key</var>] does not <a for=map>exist</a>, then set
+ <var>shed</var>[<var>key</var>] to the result of running <a>create a storage shelf</a> with
```

Comment:
"the result of creating a storage shelf" perhaps?

Reply:
That would be less locally consistent.